### PR TITLE
feat(medusa): SC service

### PR DESCRIPTION
**Pull request by pKorsholm**
**What**
- Add sales channel service with dummy methods
- Add api index file with empty router
- Add integration testing file skeleton with sales channel flag set
- Create types for sales channels (to be change if needed)

Fixes CORE-275

### CODE REVIEW

**Review by fPolic — APPROVED** [review]

File: integration-tests/api/__tests__/admin/sales-channels.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const path = require("path")
+const {} = require("@medusajs/medusa")
```
Comment (adrien2p) [inline integration-tests/api/__tests__/admin/sales-channels.js:2]: todo: rm (unless there is a specific reason to do that :))

File: integration-tests/api/__tests__/admin/sales-channels.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const { useApi } = require("../../../helpers/use-api")
+const { useDb } = require("../../../helpers/use-db")
+const adminSeeder = require("../../helpers/admin-seeder")
+const {
+  default: startServerWithEnvironment,
+} = require("../../../helpers/start-server-with-environment")
```
Comment (adrien2p) [inline integration-tests/api/__tests__/admin/sales-channels.js:9]: suggestion: `const startServerWithEnvironment = require("../../../helpers/start-server-with-environment")`

File: packages/medusa/src/api/routes/admin/sales-channels/index.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  sales_channel: SalesChannel
+}
+
+export type AdminSalesChannelDeleteRes = DeleteResponse
+
+export type AdminSalesChannelListRes = PaginatedResponse & {
+  sales_channels: SalesChannel[]
+}
+
+// export * from './'
```
Comment (adrien2p) [inline packages/medusa/src/api/routes/admin/sales-channels/index.ts:37]: todo: Think that we can remove that no?

**Review by adrien2p — APPROVED** [review]
LGTM

**Review by adrien2p — APPROVED** [review]
LGTM 🎉

**Pull request merged**